### PR TITLE
Update configuration.rst to include the `organize_migrations` setting.

### PR DESCRIPTION
**Pull request by bfoosness**
|      Q       |   A
|------------- | -----------
| Type         | documentation

### CODE REVIEW

File: docs/en/reference/configuration.rst [hunk 1]
```diff
@@ -39,6 +39,7 @@ Now, in the root of your project place a file named ``migrations.php``, ``migrat
 
             'all_or_nothing' => true,
             'check_database_platform' => true,
+            'organize_migrations' => 'year',
```
Comment (goetas) [inline docs/en/reference/configuration.rst]: by putting here `year` it seems that is the default....  does it make sense to say that the default is null and to list the possible values?

Comment (bfoosness) [reply docs/en/reference/configuration.rst]: That seems reasonable, but at least until https://github.com/doctrine/migrations/pull/1075 is merged, I think passing null will trigger an error?

For what it's worth, the table below that describes each setting does specify that the default value is false, and the description lists the two acceptable values.

Comment (bfoosness) [reply docs/en/reference/configuration.rst]: I can confirm that a value of null will trigger an error in 3.0.1. Once #1075 is merged, `false` would be the proper default value.

Comment (bfoosness) [reply docs/en/reference/configuration.rst]: `none` is now the default value per #1079.

**Review by greg0ire — APPROVED** [review]

**Review by goetas — APPROVED** [review]

**Pull request merged**